iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/RoutePreviewStatus/TransportRoutePreviewStatus.swift

```diff
@@ -42,6 +42,9 @@ final class TransportRoutePreviewStatus: SolidTouchView {
     navigationInfo = info
     etaLabel.attributedText = info.estimate
     stepsCollectionView.steps = info.transitSteps
+
+    // TODO: even when stepsCollectionView is hidden it still takes space in layout
```

Comment:
https://stackoverflow.com/questions/57655129/ios-autolayout-how-to-show-hide-a-view-including-its-margins

Reply:
There was a problem with panel min height = 80. I changed constraint to have height >= 40 and no more empty space ))